eng/pipelines/runtime.yml

```diff
@@ -1878,36 +1878,29 @@ extends:
                 eq(stageDependencies.EvaluatePaths.evaluate_paths.outputs['SetPathVars_mono_excluding_wasm.containsChange'], true),
                 eq(variables['isRollingBuild'], true))
 
-      #
-      # Sourcebuild legs
-      # We have 3 important legs for source-build:
-      # - Centos.8 (ensures that known non-portable RID is working)
-      # - Linux-x64 portable (used for dependency flow and downstream PR verification)
-      # - Banana.24 - Non-existent RID to ensure we don't break RIDs we don't know about.
-      #
-      # Running all of these everywhere is wasteful. Run Banana.24 and CentOS.8 in rolling CI,
-      # Run Linux-x64 in PR.
-
-      - template: /eng/pipelines/common/platform-matrix.yml
-        parameters:
-          jobTemplate: /eng/pipelines/common/global-build-job.yml
-          buildConfig: Release
-          helixQueueGroup: pr
-          platforms:
-          - SourceBuild_centos8_x64
-          jobParameters:
-            nameSuffix: centos8SourceBuild
-            timeoutInMinutes: 95
-            condition: eq(variables['isRollingBuild'], true)
-
-      - template: /eng/pipelines/common/platform-matrix.yml
-        parameters:
-          jobTemplate: /eng/pipelines/common/global-build-job.yml
-          buildConfig: Release
-          helixQueueGroup: pr
-          platforms:
-          - SourceBuild_banana24_x64
-          jobParameters:
-            nameSuffix: banana24SourceBuild
-            timeoutInMinutes: 95
-            condition: eq(variables['isRollingBuild'], true)
+    - stage: SourceBuild
+      displayName: Source Build Validation
+      dependsOn: []
+      condition: eq(variables['isRollingBuild'], true)
+      jobs:
+        #
+        # Sourcebuild legs
+        # We have 3 important legs for source-build:
+        # - Centos.8 (ensures that known non-portable RID is working)
+        # - Linux-x64 portable (used for dependency flow and downstream PR verification)
+        # - Banana.24 - Non-existent RID to ensure we don't break RIDs we don't know about.
+        #
+        # Running all of these everywhere is wasteful. Run Banana.24 and CentOS.8 in rolling CI,
+        # Run Linux-x64 in PR.
+        - template: /eng/common/templates/jobs/source-build.yml
+          parameters:
+            platforms:
+              - name: CentOS8
+                targetRid: centos.8-x64
```

Comment:
Isn't CentOS EOL soon?

Reply:
Yes, but I wanted to preserve existing behavior in this PR to limit possible issues.